Assets/Scripts/ECS/Animal/Needs/SexualUrgesSystem.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Unity.Entities;
+
+namespace Ecosystem.ECS.Animal.Needs
+{
+    /// <summary>
+    /// System for increasing the sexual urges of an animal
+    /// </summary>
+    public class SexualUrgesSystem : SystemBase
+    {
+        private EndSimulationEntityCommandBufferSystem m_EndSimulationEcbSystem;
+
+        protected override void OnCreate()
+        {
+            m_EndSimulationEcbSystem = World
+                .GetOrCreateSystem<EndSimulationEntityCommandBufferSystem>();
+        }
+
+        protected override void OnUpdate()
+        {
+            var commandBuffer = m_EndSimulationEcbSystem.CreateCommandBuffer().ToConcurrent();
+
+            float deltaTime = Time.DeltaTime;
+
+            Entities.ForEach((Entity entity, int entityInQueryIndex,
+                ref SexualUrgesData sexualUrgesData) =>
+            {
+                sexualUrgesData.Urge -= deltaTime / 1000.0f;
```

Comment:
```suggestion
                sexualUrgesData.Urge += deltaTime / 1000.0f;
```
Jag tycker att det är ok om den här "status bar":en fungerar omvänt mot hunger/thirst. Det känns logiskt att ett djur svälter vid 0 medan sexual urge växer hela tiden.